koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -703,12 +715,30 @@ def handler(self, src, target, opts=None):
         build_tag = target_info['build_tag']
         archlist = self.getArchList(build_tag)
 
-        label_overwrites = {}
-        release_overwrite = opts.get('release')
-        if release_overwrite:
-            label_overwrites = {LABEL_DATA_MAP['RELEASE']: release_overwrite}
-        data, expected_nvr = self.checkLabels(src, label_overwrites=label_overwrites)
+        flatpak = opts.get('flatpak', False)
+        if flatpak:
+            if not osbs_flatpak_support:
+                raise koji.BuildError("osbs-client on koji builder doesn't have Flatpak support")
+            module = opts.get('module', None)
+            if not module:
+                raise koji.BuildError("Module must be specifed for a Flatpak build")
+
+            module_name, module_stream, module_version = split_module_spec(module)
 
+            data = {
+                'name': module_name,
+                'version': module_stream,
+            }
+
+            if module_version is not None:
+                data['release'] = module_version
```

Comment:
This block doesn't get covered during tests.

Reply:
OK, should be fixed now with a bit of extra complexity to also test the cases when module is not specified or invalid.